dnstable_convert.c

```diff
@@ -385,6 +385,9 @@ process_rdata_slice(Nmsg__Sie__DnsDedupe *dns, size_t i, ubuf *key, ubuf *val)
 	case WDNS_TYPE_SRV:
 		offset = 6;	/* skip SRV priority, weight, port */
 		break;
+        case WDNS_TYPE_NSEC:
+		offset = 0;	/* skip later */
```

Comment:
The "sliced" rdata representation is not necessary for NSEC, as the rdata begins with the hostname we're interested in. Slicing is only needed for rdata layouts with the hostname offset from the start (e.g., MX, SRV, etc.)

NSEC should be handled similarly to SOA, whose rdata layout also begins with the indexed hostname.